spec/lib/ihub/appointments/service_spec.rb

```diff
@@ -24,8 +24,8 @@
       it 'returns an array of appointment data' do
         VCR.use_cassette('ihub/appointments/success', VCR::MATCH_EVERYTHING) do
           response    = subject.appointments
-          appointment = response.response_data&.dig('data')&.first
-          facility    = appointment&.dig('facility_name')
+          appointment = response.appointments&.first
+          facility    = appointment.facility_name
 
           expect(facility).to be_present
```

Comment:
This tests that the `facility_name` property isn't empty, but it could be anything and still pass right? Would it be worth matching against the value in the cassette?